recovery/bindata/upgrade-recovery.sh

```diff
@@ -245,17 +245,17 @@ function take_backup {
     cat /etc/tmpfiles.d/* | sed 's/#.*//' | awk '{print $2}' | grep '^/etc/' | sed 's#^/etc/##' > ${BACKUP_DIR}/etc.exclude.list
     echo '.updated' >> ${BACKUP_DIR}/etc.exclude.list
     echo 'kubernetes/manifests' >> ${BACKUP_DIR}/etc.exclude.list
-    with_retries 3 1 cp -Ra /etc/ ${BACKUP_DIR}/etc/
+    with_retries 3 1 cp -Ra /etc/ ${BACKUP_DIR}/
     if [ $? -ne 0 ]; then
         fatal "Failed to backup /etc"
     fi
 
-    with_retries 3 1 cp -Ra /usr/local/ ${BACKUP_DIR}/usrlocal/
+    with_retries 3 1 cp -Ra /usr/local/ ${BACKUP_DIR}/
```

Comment:
As an FYI, this was usrlocal as this is the real dirname under /ostree, and /usr/local is itself a symlink to /var/usrlocal ... but this should be fine